### PR TITLE
Add README.md to workflows for CI implementation detail.

### PR DESCRIPTION
**Pull request by mshima**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Entities
+
+Tested across others workflows.
+
+Features and technologies:
+
+- dto
+- filtering
+- fields/relationships validations
+- custom id.
```
Comment (qmonmert) [inline .github/workflows/README.md:73]:
```diff
+ custom id
- custom id.
```

File: .github/workflows/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- pagination.
```
Comment (qmonmert) [inline .github/workflows/README.md:74]:
```diff
+ pagination
- pagination.
```

**Comment by mshima** [comment]
Converted to draft, since implementation details should be discussed.

File: .github/workflows/README.md (hunk 2 above)
Comment (mshima) [inline .github/workflows/README.md:74]:
```suggestion
- custom id
- pagination
```

**Pull request closed without merging**